File: Java/Utility.java
import java.util.Scanner;

public class Utility {

	public static String input() {
		Scanner s = new Scanner(System.in);
		String text=s.next();
		s.close();
		return text;
	}

	public static void println(String ...args) {
		for (String s:args) System.out.println(s);
	}

	public static String nl(int lines) {
		String t="";
		for (int n=0;n<lines;n++) t+=System.lineSeparator();
		return t;
	}

	public static void pnl(int lines) {
		for (int m=0;m<n;m++) System.out.println();
	}

}

File: Rust/Utility.rs
#[macro_export]
macro_rules! use_util {
	() => {
		use crate::utility;
		use utility::*;
		use crate::p;
	};
}

extern crate regex;
use regex::Regex;

pub fn read() -> String {
	let mut s = String::new();
	std::io::stdin().read_line(&mut s).ok();
	s.trim_end().to_string()
}

pub fn pnl(lines:isize) {
	for _ in 0..lines { println!(""); }
}

pub fn clean(text:String) -> String {
	let trimmer = Regex::new("^\\t+").unwrap();
	return trimmer.replace_all(&text,"").to_string();
}

#[macro_export]
macro_rules! p {
	($text:expr,$lines:expr) => {
		println!("{}",clean($text.to_string()));
		pnl($lines);
	};
}